templates/web.conf.erb

```diff
@@ -1,10 +1,10 @@
 # The externally visible URL for accessing this instance of IRIDA. This key is
 # used by the e-mailer when sending out e-mail notifications (password resets,
 # for example) and embeds this URL directly in the body of the e-mail.
-<% if @use_ssl %>
-server.base.url=https://<%= @irida_ip_addr %>/<%= @irida_url_path %>
+<% if @use_ssl && @force_ssl  %>
+server.base.url=https://<%= @server_base_url %>/<%= @irida_url_path %>
```

Comment:
We have to check both **use_ssl** and **force_ssl** is set before indicating that we are only using https. We have these flags so we can slowly transition old IRIDA syncing, irida linker ,galaxy importer to https. When all are done, will remove force_ssl flag.